tests/python3.8/boto_tests/unit/test_tokens.py

```diff
@@ -286,6 +287,10 @@ async def test_sso_token_provider_refresh(test_case):
     cache_key = "d033e22ae348aeb5660fc2140aec35850c4da997"
     token_cache = {}
 
+    # deepcopy the test case so the test can be parametrized against the same
+    # test case w/ aiohttp & httpx
+    test_case = deepcopy(test_case)
```

Comment:
oh man, `parametrize` should be returning a frozendict.  That's crazy of botocore changing global state in a test, we inherited this code from botocore: https://github.com/boto/botocore/blob/develop/tests/unit/test_tokens.py#L26

Reply:
yeah it sounds like botocore should get a PR